lib/rex/powershell/command.rb

```diff
@@ -340,9 +363,9 @@ def self.cmd_psh_payload(pay, payload_arch, template_path, opts = {})
       command_args[:command] = final_payload
     end
 
-    psh_command = generate_psh_command_line(command_args)
+    psh_command = opts[:exec_in_place] ? "#{command_args[:command]}" : generate_psh_command_line(command_args)
```

Comment:
seems like a legitimate use here - it's only being used for a single assignment with no side-effects